src/core/main.c

```diff
@@ -582,6 +583,53 @@ static int config_parse_cpu_affinity2(
         return 0;
 }
 
+static int config_parse_numa_policy(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        int r;
+        NUMAPolicy *policy = data;
+
+        assert(policy);
+
+        r = numa_policy_type_from_string(rvalue);
+        if (r < 0)
+                return log_warning_errno(r, "Failed to parse NUMA memory policy: %m");
+
+        policy->type = r;
+
+        return 0;
+}
+
+static int config_parse_numa_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        NUMAPolicy *policy = data;
+
+        assert(policy);
+
+        (void) parse_cpu_set_extend(rvalue, &policy->nodes, true, unit, filename, line, lvalue);
+
+        return 0;
+}
```

Comment:
these two functions we already basically have, in load-fragment.c. please reuse.